src/aat/resources/features/F-027/F-027.feature

```diff
@@ -76,6 +76,19 @@ Scenario: must return appropriate negative response for a user not having a prof
     And the response [has the 403 return code],
     And the response has all other details as expected.
 
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-039
+Scenario: must return appropriate negative response for invalid caseType
+
+    Given a user with [an active profile in CCD],
+
+     When a request is prepared with appropriate values,
+      And the request [provides an invalid case type id],
+      And it is submitted to call the [Fetch a case for display for Case Worker] operation of [CCD Data Store],
+
+     Then a negative response is received,
+      And the response [has the 400 return code].
```

Comment:
A better option for this line could be:
`And the response has all other details as expected.` (pls check the exact spelling/words)
This way, the scenario would read less techie from a Business Analyst's PoV. Remember that these lines would have ideally been scripted by a BA and they would ideally describe the requirement in terms of a negative response, not necessarily telling the http code to return. The http code to return is defined by low level design and is embedded into the test data already. Pls apply this to the scenarios edited in this PR.